index.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>CSD120 - Calvin Lane</title>
 </head>
-<"Hello, World!">
-    
+<body>
+    Hello, World!
 </body>
```

Comment:
When adding content to an element, don't replace the opening tag name, just place the content between the opening and closing tags.